imap/command.c

```diff
@@ -1240,8 +1240,6 @@ const char *imap_cmd_trailer(struct ImapAccountData *adata)
  * Also, handle untagged responses.
```

Comment:
The `@retval`s above need changing to match the new return codes.

Reply:
done